File: app/src/main/java/org/fattili/easypopup_demo/java/CardTestPop.java
package org.fattili.easypopup_demo.java;

import android.app.Activity;
import android.view.Gravity;
import android.view.View;
import android.view.ViewGroup;

import org.fattili.easypopup.view.card.CardPopup;
import org.fattili.easypopup_demo.R;
import org.jetbrains.annotations.Nullable;

/**
 * 2021/2/22
 *
 * @author dengzhenli
 */
public class CardTestPop extends CardPopup {


    public CardTestPop(@Nullable Activity activity) {
        super(activity);
    }

    public CardTestPop(@Nullable Activity activity, int gravity, int width, int height) {
        super(activity, gravity, width, height);
    }

    @Override
    public boolean outClickable() {
        return true;
    }

    @Override
    public int getContentLayoutId() {
        return R.layout.card_pop_test;
    }

}

File: app/src/main/java/org/fattili/easypopup_demo/kt/CardTestPop.kt
package org.fattili.easypopup_demo.kt

import android.app.Activity
import android.util.Log
import org.fattili.easypopup.view.card.CardPopup
import org.fattili.easypopup_demo.R

/**
 * 2021/2/22
 *
 * @author dengzhenli
 */
class CardTestPop : CardPopup {
    constructor(activity: Activity) : super(activity) {}
    constructor(activity: Activity, gravity: Int, width: Int, height: Int) : super(
        activity,
        gravity,
        width,
        height
    ) {
    }



    override fun getContentLayoutId(): Int {
        return R.layout.card_pop_test
    }

    private val TAG = "CardTestPop"

}